test/unit_tests/braket/circuits/test_ascii_circuit_diagram.py

```diff
@@ -872,3 +872,17 @@ def __init__(self):
         "T  : |    0    |   1    |   2   |   3   |   4   |",
     )
     _assert_correct_diagram(circ, expected)
+
+
+def test_measure():
+    circ = Circuit().h(0).cnot(0, 1).measure([0])
+    expected = (
+        "T  : |0|1|2|",
+        "            ",
+        "q0 : -H-C-M-",
+        "        |   ",
+        "q1 : ---X---",
+        "",
+        "T  : |0|1|2|",
+    )
+    _assert_correct_diagram(circ, expected)
```

Comment:
can we also test measure for multiple qubits? does it have a line connecting the Ms? do we want this? I'm thinking maybe, otherwise they may be staggered to different time steps

Reply:
Added a test for multiple qubit measurements here and in test_unicode_circuit_diagram